actions/terraform-checks/action.yml

```diff
@@ -1,6 +1,13 @@
 ---
 name: "Terraform Checks"
 description: "Perform common Terraform checks"
+
+# Can be overriden by the developer when using this action
+inputs:
+  terraform_version:
+    description: "The version of Terraform to use"
+    default: "1.94"
```

Comment:
I've set this to the newest version as default - but pinned. This prevents version from changing around ONA.